src/core/generators/imports.ts

```diff
@@ -63,12 +64,18 @@ export const generateMutatorImports = (
       const importDefault = mutator.default
         ? `${mutator.name}${
             mutator.hasErrorType
```

Comment:
what do you think of doing something like this mutator.hasErrorType || mutator.bodyTypeName and then handling rest inside it?

Reply:
Indeed it will be clearer 👍 
I did the change, let me know :)